AutoCollection/Statsbeat.ts

```diff
@@ -232,17 +275,23 @@ class Statsbeat {
     private _getResourceProvider(callback: () => void) {
         // Check resource provider
         let waiting: boolean = false;
+        this._resourceIdentifier = Constants.StatsbeatResourceProvider.unknown;
         if (process.env.WEBSITE_SITE_NAME) { // Web apps
             this._resourceProvider = Constants.StatsbeatResourceProvider.appsvc;
+            this._resourceIdentifier = process.env.WEBSITE_SITE_NAME;
```

Comment:
@heyams  Current specs says `WEBSITE_SITE_NAME/WEBSITE_HOME_STAMPNAME` for `rpid`. Does this mean the literal string (with the slash concatenating the two), or choose `WEBSITE_SITE_NAME` if available, otherwise `WEBSITE_HOME_STAMPNAME`?

Reply:
I expect WEBSITE_SITE_NAME/WEBSITE_HOME_STAMPNAME.  if stampName is empty, it will be "{WEBSITE_SITE_NAME}/".  yes.